src/controllers/calendars.cr

```diff
@@ -9,6 +9,11 @@ class Calendars < Application
   @[AC::Route::Filter(:around_action)]
   Application.add_request_queue
 
+  @[AC::Route::Filter(:before_action)]
+  private def ensure_tenant
+    raise Error::NotImplemented.new("domain does not have a tenant configured") unless current_tenant
```

Comment:
we can actually just call `current_tenant` as it will raise appropriately if a tenant isn't configured